internal/runners/clean/run_lin_mac.go

```diff
@@ -158,15 +157,30 @@ func removeEmptyDir(dir string) error {
 }
 
 func cleanInstallDir(dir string) error {
+	stateExec, err := installation.NewExec(installation.StateExec)
+	if err != nil {
+		return locale.WrapError(err, "err_state_exec")
+	}
+
+	serviceExec, err := installation.NewExec(installation.ServiceExec)
+	if err != nil {
+		return locale.WrapError(err, "err_service_exec")
+	}
+
+	trayExec, err := installation.NewExec(installation.TrayExec)
+	if err != nil {
+		return locale.WrapError(err, "err_tray_exec")
+	}
```

Comment:
Perhaps we could have a single `installation.Executables()` function which enumerates ALL executables. The we only need a single error assertion.

Reply:
Good idea!